frontend/src/components/Profile.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react"
+import styled from "styled-components/macro"
+import { user, userId, logout, setErrorMessage } from "../reducers/user"
+import { useDispatch, useSelector } from 'react-redux'
+
+
+const URL = "https://signinprojecttechnigo.herokuapp.com/users/"
+
+const ProfileWrapper = styled.div`
+background-color: #FDE7EA;
+padding: 20px;
+margin-bottom: 20px;
+width: 45%;
+`
+
+const Button = styled.button`
+background-color: #FF7C98;
+color: #FFFF;
+font-family: 'Poppins', sans-serif;
+font-weight: 700;
+text-transform: uppercase; 
+outline: none;
+border: none;
+margin: 20px;
+`
+
+export const Profile = ({ accsessToken }) => {
+  const dispatch = useDispatch();
+  const accessToken = useSelector((store) => store.user.login.accessToken);
+  const userId = useSelector((store) => store.user.login.userId);
+  const secretMessage = useSelector((store) => store.user.login.secretMessage);
+  const errorMessage = useSelector((store) => store.user.login.errorMessage);
+
+  return (
+    <ProfileWrapper>
+      {errorMessage && <h4>Error Message : {`${errorMessage}`}</h4>}
```

Comment:
I'm guessing you'd want this in the `SignUp` or `LogIn` component, because the `Profile` component only gets mounted if there's an accessToken.